fluentui_core/src/main/java/com/microsoft/fluentui/theme/token/controlTokens/AvatarCarouselTokens.kt

```diff
@@ -0,0 +1,93 @@
+package com.microsoft.fluentui.theme.token.controlTokens
+
+import android.os.Parcelable
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.Dp
+import com.microsoft.fluentui.theme.FluentTheme
+import com.microsoft.fluentui.theme.token.*
+import kotlinx.parcelize.Parcelize
+
+enum class AvatarCarouselSize{
+    Medium,
+    Large
+}
+enum class TextType {
+    Text,
+    SubText
+}
+
+@Parcelize
```

Comment:
Create a AvatarCarouselInfo extending ControlInfo object just like done for other control & pass that as arguments in function.